src/libstd/sys_common/mod.rs

```diff
@@ -33,8 +33,18 @@ pub mod at_exit_imp;
 #[cfg(feature = "backtrace")]
 pub mod backtrace;
 pub mod condvar;
+#[cfg(any(unix,
+          target_os = "redox",
+          all(target_vendor = "fortanix", target_env = "sgx")))]
+pub mod ffi_bytes;
```

Comment:
Could this be folded into the `os_str_bytes` module below? Each individual ext module would then `pub use` the relevant traits directly.